scripts/core/editor3/reducers/table.jsx

```diff
@@ -152,6 +152,7 @@ const processCells = (state, fn) => {
         return state;
     }
 
+    const entityDataHasChanged = true;
```

Comment:
Same with this one, please put it after line 164. If it stays here, it's not quite accurate because the entity data has not _yet_ changed.